web/api/netdata-swagger.yaml

```diff
@@ -35,137 +34,219 @@ paths:
+      summary: Get a list of all charts available at the server
+      description: >-
+        The charts endpoint returns a summary about all charts stored in the
+        netdata server.
       responses:
         '200':
-          description: 'An array of charts'
+          description: An array of charts
           schema:
             type: array
             items:
               $ref: '#/definitions/chart_summary'
   /chart:
     get:
-      summary: 'Get info about a specific chart'
-      description: 'The Chart endpoint returns detailed information about a chart.'
+      summary: Get info about a specific chart
+      description: The Chart endpoint returns detailed information about a chart.
       parameters:
         - name: chart
           in: query
-          description: 'The id of the chart as returned by the /charts call.'
+          description: The id of the chart as returned by the /charts call.
           required: true
           type: string
-          format: 'as returned by /charts'
-          default: 'system.cpu'
+          format: as returned by /charts
+          default: system.cpu
       responses:
         '200':
-          description: 'A javascript object with detailed information about the chart.'
+          description: A javascript object with detailed information about the chart.
           schema:
             $ref: '#/definitions/chart'
+        '400':
+          description: No chart id was supplied in the request
         '404':
-          description: 'No chart with the given id is found.'
+          description: No chart with the given id is found.
   /alarm_variables:
     get:
-      summary: 'List variables available to configure alarms for a chart'
-      description: 'Returns the basic information of a chart and all the variables that can be used in alarm and template health configurations for the particular chart or family'
+      summary: List variables available to configure alarms for a chart
+      description: >-
+        Returns the basic information of a chart and all the variables that can
+        be used in alarm and template health configurations for the particular
+        chart or family
       parameters:
         - name: chart
           in: query
-          description: 'The id of the chart as returned by the /charts call.'
+          description: The id of the chart as returned by the /charts call.
           required: true
           type: string
-          format: 'as returned by /charts'
+          format: as returned by /charts
       responses:
         '200':
-            description: 'A javascript object with information about the chart and the available variables'
-            schema:
-              $ref: '#/definitions/alarm_variables'
+          description: >-
+            A javascript object with information about the chart and the
+            available variables
```

Comment:
You are missing `.` at the end, I know the original had the same problem, but I saw that there are descriptions with dots and others without, and I arrived in the conclusion that the best would be everything with dot at the end.

Reply:
I saw the same thing when I was writing the strings. In the end I went with dot-less to try to be consistent. I'll do a grep / count over the json and see which is more common.

Reply:
```
grep description web/api/netdata-swagger.json | wc -l
     183
grep description web/api/netdata-swagger.json | grep '\."' | wc -l
     105
```
Yes, you are correct - the dot at the end is grammatically correct and it is the more common choice. I'll put dots on all the descriptions.

Reply:
Fixed.